src/PublicApiClient.php

```diff
@@ -119,4 +119,24 @@ public function getCustomersClient(): CustomersClient
             ->setUrl($this->url)
             ->setApiKey($this->apiKey);
     }
+
+    /**
+     * @return Billing\StatementsClient
+     */
+    public function getBillingStatementsClient(): Billing\StatementsClient
+    {
+        return (new Billing\StatementsClient($this->client))
```

Comment:
import the dependency with a use and remove the `Billing\` part

Reply:
Preferences and Statements are words common enough to be misleading without the namespace
Is it okay to leave it this way ? or do you want to remove the namespace just for stylistic issues ?